sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-messaging-servicebus/README.md

```diff
@@ -28,13 +28,13 @@ dependency `azure-spring-cloud-messaging`.
     <dependency>
       <groupId>com.azure.spring</groupId>
       <artifactId>azure-spring-cloud-starter-servicebus</artifactId>
-      <version>2.3.0</version>
+      <version>2.3.0</version> <!-- {x-version-update;com.azure.spring:azure-spring-cloud-starter-servicebus;current} -->
```

Comment:
I'm not sure whether this is the correct way to add the tag, please refer to this  https://github.com/Azure/azure-sdk-for-java/blob/master/CONTRIBUTING.md#tooling-version-files-and-marker-tags
![image](https://user-images.githubusercontent.com/31124698/113105256-fa952d80-9233-11eb-845c-a606e6a9a200.png)

Reply:
get